src/megaapi_impl.cpp

```diff
@@ -13847,6 +13971,54 @@ void MegaApiImpl::sendPendingRequests()
             }
 			break;
 		}
+        case MegaRequest::TYPE_RESTORE:
+        {
+            Node *version = client->nodebyhandle(request->getNodeHandle());
+            if (!version)
+            {
+                e = API_ENOENT;
+                break;
+            }
+
+            if (version->type != FILENODE || !version->parent || version->parent->type != FILENODE)
+            {
+                e = API_EARGS;
+                break;
+            }
+
+            Node *current = version;
+            while (current->parent && current->parent->type == FILENODE)
```

Comment:
I'm not sure if it would be useful for apps. They will probably get the versions using `getVersions` and the returned list already contains the current version. Anyway, we can add it as soon as it's needed.

Reply:
Fine. Let's add it only if we see the need :)